platform/security/core/security-core-impl/src/main/java/ddf/security/service/impl/SecurityAssertionStore.java

```diff
@@ -48,26 +47,24 @@
     public static SecurityAssertion getSecurityAssertion(Message message) {
         if (message != null) {
             TokenStore tokenStore = getTokenStore(message);
-            Principal principal = (Principal) message.get(WSS4JInInterceptor.PRINCIPAL_RESULT);
```

Comment:
They actually removed that property completely and no longer put the principal in the message. The code is still functionally the same, we just now have to loop through the results to see if the SAMLTokenPrincipal exists. https://github.com/apache/cxf/commit/fc78cd31869a4e580c4d2a9e7864ae6dae8c311b#diff-d5cedb5c4d36fd555c39401f90a25359L89